components/svg/UiUx.js

```diff
@@ -0,0 +1,127 @@
+export default function UiUx() {
+	return (
+		<svg
+			width='125'
+			height='79'
```

Comment:
You don't want a fixed width/height.  What is UiUx? UI = user interface UX = User experience This defines an experience or a view... not really an icon.